pkg/network/server_test.go

```diff
@@ -4,29 +4,33 @@ import (
 	"net"
 	"testing"
 
+	"github.com/nspcc-dev/neo-go/pkg/network/capability"
 	"github.com/nspcc-dev/neo-go/pkg/network/payload"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
 func TestSendVersion(t *testing.T) {
 	var (
-		s = newTestServer(t)
+		s = newTestServer(t, ServerConfig{Port: 3000, UserAgent: "/test/"})
```

Comment:
NACK. This port might be unavailable on testing machine. It should either use 0 for dynamic port of use non-TCP transport (like it was done before).